connect.go

```diff
@@ -336,7 +336,11 @@ func (c *HCPConnectCommand) listClusters(organizationID string, projectID string
 
 	default:
 		cluster := clustersResp.GetPayload().Clusters[0]
-		if *cluster.State != hcpvsm.HashicorpCloudVault20201125ClusterStateRUNNING {
+		if *cluster.State == hcpvsm.HashicorpCloudVault20201125ClusterStateLOCKED || *cluster.State == hcpvsm.HashicorpCloudVault20201125ClusterStateLOCKING {
+			return "", errors.New("cluster is locked")
+		} else if *cluster.State == hcpvsm.HashicorpCloudVault20201125ClusterStateCREATING {
+			return "", errors.New("cluster is still being created")
+		} else if *cluster.State != hcpvsm.HashicorpCloudVault20201125ClusterStateRUNNING {
```

Comment:
I attempted to refactor this from if/else if to a switch/case. Unfortunately, it looks roughly the same since we cannot switch based on the value of `*cluster.State`. I can create a variable for the state though to prevent multiple dereferences.